app.js

```diff
@@ -0,0 +1,21 @@
+const browserObject = require('./browser');
+const scraperController = require('./pageController');
+
+//Start the browser and create a browser instance
+let browserInstance = browserObject.startBrowser();
+
+// Pass the browser instance to the scraper controller
+scraperController(browserInstance)
+
+// const request= require("request-promise")
```

Comment:
if this code is not of any use...why it is still there?

Reply:
I removed it.